src/perfrecordssh.h

```diff
@@ -0,0 +1,55 @@
+/*
+    SPDX-FileCopyrightText: Lieven Hey <lieven.hey@kdab.com>
+    SPDX-FileCopyrightText: 2022 Klarälvdalens Datakonsult AB, a KDAB Group company, info@kdab.com
+
+    SPDX-License-Identifier: GPL-2.0-or-later
+*/
+
+#pragma once
+
+#include "perfrecord.h"
+
+class QFile;
+
+class PerfRecordSSH : public PerfRecord
```

Comment:
please add some basic documentation on how this is implemented. basically just something that says we overload a bunch of API to disable features and then use an external ssh process to communicate with another machine to remote control the perf session there